packages/bundle-size/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "extends": ["//"],
+  "tasks": {
+    "bundle-size": {
```

Comment:
For tasks specific to a package, it seems most understandable to let it extend turbo.json.